lib/go/templates/marketV3_templates.go

```diff
@@ -5,142 +5,144 @@ import (
+	changePriceV3Filename         = "marketV3/change_price.cdc"
+	changePercentageV3Filename    = "marketV3/change_percentage.cdc"
+	changeOwnerReceiverV3Filename = "marketV3/change_receiver.cdc"
+	purchaseV3Filename            = "marketV3/purchase_moment.cdc"
+	mintAndPurchaseV3Filename     = "marketV3/mint_and_purchase.cdc"
+	upgradeSaleFilename           = "marketV3/upgrade_sale.cdc"
+
+	purchaseBothMarketsFilename = "marketV3/purchase_both_markets.cdc"
 
 	// scripts
-	getSalePriceV2Filename      = "marketV2/scripts/get_sale_price.cdc"
-	getSalePercentageV2Filename = "marketV2/scripts/get_sale_percentage.cdc"
-	getSaleLengthV2Filename     = "marketV2/scripts/get_sale_len.cdc"
-	getSaleSetIDV2Filename      = "marketV2/scripts/get_sale_set_id.cdc"
+	getSalePriceV3Filename      = "marketV3/scripts/get_sale_price.cdc"
+	getSalePercentageV3Filename = "marketV3/scripts/get_sale_percentage.cdc"
+	getSaleLengthV3Filename     = "marketV3/scripts/get_sale_len.cdc"
+	getSaleSetIDV3Filename      = "marketV3/scripts/get_sale_set_id.cdc"
 )
 
-// This contains template transactions for the second version of the Top Shot
+// This contains template transactions for the third version of the Top Shot
 // marketplace, which uses a capability to access the owner's moment collection
 // instead of storing the moments in the sale collection directly
 
 // GenerateCreateSaleScript creates a cadence transaction that creates a Sale collection
 // and stores in in the callers account published
-func GenerateCreateSaleV2Script(env Environment) []byte {
-	code := assets.MustAssetString(transactionsPath + createSaleV2Filename)
+func GenerateCreateSaleV3Script(env Environment) []byte {
+	code := assets.MustAssetString(transactionsPath + createSaleV3Filename)
 
 	return []byte(replaceAddresses(code, env))
 }
 
 // GenerateStartSaleScript creates a cadence transaction that starts a sale by depositing
 // an NFT into the Sale Collection with an associated price
-func GenerateStartSaleV2Script(env Environment) []byte {
-	code := assets.MustAssetString(transactionsPath + startSaleV2Filename)
+func GenerateStartSaleV3Script(env Environment) []byte {
+	code := assets.MustAssetString(transactionsPath + startSaleV3Filename)
 
 	return []byte(replaceAddresses(code, env))
 }
 
 // GenerateCreateAndStartSaleScript creates a cadence transaction that creates a Sale collection
 // and stores in in the callers account, and also puts an NFT up for sale in it
-func GenerateCreateAndStartSaleV2Script(env Environment) []byte {
-	code := assets.MustAssetString(transactionsPath + createAndStartSaleV2Filename)
+func GenerateCreateAndStartSaleV3Script(env Environment) []byte {
+	code := assets.MustAssetString(transactionsPath + createAndStartSaleV3Filename)
 
 	return []byte(replaceAddresses(code, env))
 }
 
-// GenerateCancelSaleV2Script creates a cadence transaction that starts a sale by depositing
+// GenerateCancelSaleV3Script creates a cadence transaction that starts a sale by depositing
```

Comment:
Just to double check - we would use this now over the `GenerateWithdrawFromSaleScript`  because we no longer need a a withdraw in order to delist a moment, right? (Do we need to update the comment for this one?)

Reply:
yep, I'll change the comment